### PR TITLE
Spike/refactor translation mp service

### PR DESCRIPTION
**Pull request by IngageMaddoxT**

### CODE REVIEW

**Comment by IngageMaddoxT** [comment]
@danrye Please review pull request.  I combined the Update, Create, Delete Page Services into a single file.  All using the Platform Util now to remove redundant code.  Properly updated namespaces.

Have a good weekend.

File: MinistryPlatform.Translation.Test/TranslationTests.cs [hunk 1]
```diff
@@ -174,7 +174,7 @@ public void ShouldNotDeleteResponse()
             var responseId = OpportunityService.RespondToOpportunity(token, opportunityId, comment);
 
             //Try to delete Response, should fail
-            Assert.Throws<FaultException<ExceptionDetail>>(() => DeletePageRecordService.DeleteRecord(382, responseId, null, token));
+            Assert.Throws<FaultException<ExceptionDetail>>(() => MinistryPlatformService.DeleteRecord(382, responseId, null, token));
```
Comment (danrye) [inline MinistryPlatform.Translation.Test/TranslationTests.cs]: We should update 382 to use value from ConfigurationManager.  Not really part of this PR, I know.

File: crds-angular/Services/AccountService.cs [hunk 2]
```diff
@@ -74,7 +74,7 @@ public AccountInfo getAccountInfo(string token)
             householdDictionary["Congregation_ID"] = Convert.ToInt32(ConfigurationManager.AppSettings["Congregation_Default_ID"]);
             householdDictionary["Household_Source_ID"] = Convert.ToInt32(ConfigurationManager.AppSettings["Household_Default_Source_ID"]);
 
-            recordId = MinistryPlatform.Translation.Services.CreatePageRecordService.CreateRecord(Convert.ToInt32(ConfigurationManager.AppSettings["Households"]), householdDictionary, token);
+            recordId = MinistryPlatform.Translation.Services.MinistryPlatformService.CreateRecord(Convert.ToInt32(ConfigurationManager.AppSettings["Households"]), householdDictionary, token);
```
Comment (danrye) [inline crds-angular/Services/AccountService.cs]: Should be able to remove MinistryPlatform. Translation.Services from each of these blocks.  Again not part of this PR, but while we're here.

File: MinistryPlatform.Translation/Services/AuthenticationService.cs [hunk 3]
```diff
@@ -52,7 +51,7 @@ public static Boolean ChangePassword(string token, string newPassword)
             {
                 var record = GetPageRecordService.GetRecordsDict(Convert.ToInt32(ConfigurationManager.AppSettings["ChangePassword"]), token).Single();
```
Comment (danrye) [inline MinistryPlatform.Translation/Services/AuthenticationService.cs]: Shouldn't we remove GetPageRecordService also? @IngageMaddoxT

**Comment by danrye** [comment]
@IngageMaddoxT I made some changes, please review.

**Comment by IngageMaddoxT** [comment]
@danrye I reviewed things.  Looks like you addressed all of your concerns.  I cleaned up the Lookup service a bit and merged dev back into the branch.  Please review again.  Thanks for your help.
